components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/idp/sso/SSOConfiguratorUtil.java

```diff
@@ -299,9 +299,9 @@ private static String getGatewayUrl(WebApp app) {
         String tenantDomain = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain();
 
         if(!tenantDomain.equalsIgnoreCase("carbon.super")){
-            url = url + "/t/" + tenantDomain + app.getContext() + "/" + identifier.getVersion() + "/";
+            url = url + "/t/" + tenantDomain + app.getContext() + "/" ;
         }else {
-            url = url + app.getContext() + "/" + identifier.getVersion() + "/";
+            url = url + app.getContext() + "/" ;
```

Comment:
301, 303 lines are not committed by me.
302, 304 lines: fixed in commit dd67fd14b1ff8ac962430217e7c5bee152796b2a of PR https://github.com/wso2/carbon-appmgt/pull/153